spring-integration-core/src/main/java/org/springframework/integration/config/DefaultConfiguringBeanFactoryPostProcessor.java

```diff
@@ -233,7 +233,7 @@ private void registerErrorChannel() {
 
 			BeanComponentDefinition componentDefinition =
 					new BeanComponentDefinition(loggingEndpointBuilder.getBeanDefinition(),
-							IntegrationContextUtils.ERROR_LOGGER_BEAN_NAME);
+							"errorLogger", new String[] { IntegrationContextUtils.ERROR_LOGGER_BEAN_NAME });
```

Comment:
Hmm... maybe too generic a name - likely to collide with a user bean.

`default_errorLogger`
`integration_errorLogger`
`si_errorLogger`
`_si_errorLogger`
...

might be better choices.

Reply:
Well, my point is to align with the `errorChannel` indicating that this default logger is definitely tied with that global default channel.
Yes, it could collide with something in the target application, that why I'm targeting this for `5.3`, so, we have a room for some breaking changes with an appropriate Migration Guide.

Reply:
I understand your goal; but my point is that people will definitely use `errorChannel` in their configuration - particularly on gateways where they just want the error logged.

However, I doubt that anyone will explicitly reference the default error logger - hence its obscure name.

I just think it might be an unnecessary burden to break an app that already has a bean `errorLogger` that might not even be related to the Spring Integratin part of the application, especially when this change is really just to have a shorter name in the graph.